include/networkmanager.h

```diff
@@ -14,6 +14,8 @@
 #undef MS_FAILOVER_SUPPORTED
 #endif
 
+#undef MS_FAILOVER_SUPPORTED
```

Comment:
I don't think this was intentional on your end, but if it was, you might as well delete the failover code if you are going to do this.

Reply:
wasn't me